src/iv/ivgl.cpp

```diff
@@ -903,6 +937,44 @@ IvGL::paint_pixelview()
 
 
 
+void
+IvGL::paint_datawindow()
+{
+    IvImage* img = m_current_image;
+    const ImageSpec& spec(img->spec());
+
+    glDisable(GL_TEXTURE_2D);
+    glUseProgram(0);
+
+    glColor4f(0.5f, 0.5f, 0.5f, 1.f);
```

Comment:
What do other viewer apps do here? Is 0.5 grey the right choice? Should it be a color that is unusual in natural imagery, like bright magenta, so there's almost no conceivable image in which it wouldn't be clearly visible?

Reply:
Good question, I am not sure either. But I did try to use the `GL_XOR` logical operation while drawing the borders, which gives great contrast in non gray-tone images. The GrayRemapHorizontal example shows where it fails around gray pixels. 
|MtTamWest|GrayRemapHorizontal|
|:--:|:--:|
|<img width="318" alt="image" src="https://github.com/user-attachments/assets/6faeae43-be11-4dbe-a266-bd8adf3e528e">|<img width="318" alt="image" src="https://github.com/user-attachments/assets/9af0312c-a399-430e-846e-d770c2284f46">|

Nonetheless, the border is still very visible and it adapts to the viewing image without having to set a constant color.

Reply:
My bad, I didn't notice the XOR. I think that's fine. We can always revise if people complain that there are significant cases where it's unclear.

Reply:
No, not at all. I was experimenting the XOR option yesterday, so my changes weren't pushed yet. I agree we can revise this later if needed.